docs/docs/guides/labs/components/creating-new-component-types/creating-and-registering-a-component-type.md

```diff
@@ -31,7 +31,7 @@ This will add a new file to your project in the `lib` directory:
 
 This file contains the basic structure for the new component type. There are two methods that you'll need to implement:
 
-- `get_schema`: This method should return a Pydantic model that defines the schema for the component. This is the schema for the data that goes into `component.yaml`.
+- `get_schema`: This method should return a Pydantic based `ResolvableModel` that defines the schema for the component. This is the schema for the data that goes into `component.yaml`.
```

Comment:
nit: Pydantic-based? not even that confident that's right so feel free to ignore